scripts/bench/benchmarks/hacker-news/generate.js

```diff
@@ -1,16 +1,20 @@
 'use strict';
 
 const fetch = require('node-fetch');
-const { writeFileSync } = require('fs');
+const {writeFileSync} = require('fs');
 const stories = 50;
 
 async function getStory(id) {
-  const storyRes = await fetch(`https://hacker-news.firebaseio.com/v0/item/${id}.json`);
+  const storyRes = await fetch(
+    `https://hacker-news.firebaseio.com/v0/item/${id}.json`,
```

Comment:
I don't see how this could work. It should be ES6 (which doesn't support trailing commas).

Reply:
It's my fault. On previous commit, I was uncareful to ignore benchmark, so it ran prettier on those files.
I've reverted it on latest commit.